docs/getting-started/yaml.md

```diff
@@ -0,0 +1,78 @@
+# Yaml Configuration
+
+This is an interactive guide for exploring various important properties of the screwdriver.yaml configuration for projects.
+
+You can access information about properties by mousing over the property name.
+
+<div class="yaml-docs">
+
+<pre class="example">
+<a href="#workflow"><span class="key">workflow</span>:
+    - <span class="value">publish</span>
+    - <span class="key">parallel</span>:
+        - <span class="key">series</span>:
+            - <span class="value">deploy-east</span>
+            - <span class="value">validate-east</span>
+        - <span class="key">series</span>:
+            - <span class="value">deploy-west</span>
+            - <span class="value">validate-west</span></a>
+<a href="#shared"><span class="key">shared</span>:</a>
+    <a href="#environment"><span class="key">environment</span>:
+        <span class="key">NODE_ENV</span>: <span class="value">test</span></a>
+<a href="#jobs"><span class="key">jobs</span>:</a>
+    <a href="#main-job"><span class="key">main</span>:</a>
+        <a href="#image"><span class="key">image</span>: <span class="value">node:{{NODE_VERSION}}</span></a>
+        <a href="#matrix"><span class="key">matrix</span>:
+    <span class="key">NODE_VERSION</span>: <span class="value">[4,5,6]</span></a>
+        <a href="#steps"><span class="key">steps</span>:
+    <span class="key">init</span>: <span class="value">npm install</span>
+    <span class="key">test</span>: <span class="value">npm test</span></a>
+    <a href="#jobs"><span class="key">publish</span>:
+    <span class="key">image</span>: <span class="value">node:6</span>
+    <span class="key">steps</span>:
+        <span class="key">publish</span>: <span class="value">npm publish</span></a>
+    <a href="#jobs"><span class="key">deploy-west</span>:
+    <span class="key">image</span>: <span class="value">node:6</span>
+    <span class="key">environment</span>:
+        <span class="key">DEPLOY_ENV</span>: <span class="value">west</span>
+    <span class="key">steps</span>:
+        <span class="key">init</span>: <span class="value">npm install</span>
+        <span class="key">publish</span>: <span class="value">npm deploy</span></a>
+    <a href="#jobs">...</a>
+</pre>
+    <div class="yaml-side">
+        <div id="workflow" class="hidden">
+            <h4>Workflow</h4>
+            <p>Defines the order of jobs that are executed for the project. All jobs referenced by the workflow must be defined in the jobs section.</p>
+            <p>Jobs can execute in parallel or in series, and in combinations of the two, per this example. Special keywords <strong><em>parallel</em></strong> and <strong><em>series</em></strong> define the flow of the jobs. By default, the jobs in the workflow list are run in series after <em>main</em> job has completed successfully.</p>
```

Comment:
`Jobs can execute in parallel, in series, or in any combination of the two, per this example.`